test/contract.test.ts

```diff
@@ -14,6 +14,28 @@ const txContext = { inputSatoshis, tx };
 
 const jsonDescr = loadDescription('p2pkh_desc.json');
 
+
+
+describe('check constructor()', () => {
```

Comment:
P2PKH contract has explicit constructor. Add a test for a contract with implicit constructor, e.g., https://github.com/sCrypt-Inc/scryptlib/blob/master/test/fixture/cointoss.scrypt.